ext_mongodb.php

```diff
@@ -715,17 +720,43 @@ function __debugInfo() : array;
 {
 	use DenySerialization;
 
-	public function __construct(private string $code, private ?mixed $scope = NULL)
+	private $code;
+
+	public function __construct(string $code, private ?mixed $scope = NULL)
 	{
+		if ( strstr( $code, "\0" ) !== false )
+		{
+			throw new \MongoDB\Driver\Exception\InvalidArgumentException( "Code cannot contain null bytes" );
+		}
+		$this->code = $code;
```

Comment:
Aha! That syntactic sugar rings a bell, but I missed the visibility modifiers in the argument list.